docs/docs/guides/context.md

```diff
@@ -0,0 +1,131 @@
+# Enrich Apps Traces with Context and Metadata
+
+This guide will walk you through the usage of Context in Keptn.
+
+After reading this guide you will be able to:
+
+- Add metadata to application and workloads traces
+- Link traces between different versions of your application
+
+`KeptnAppContext` is a custom resource in Keptn that allows you to add metadata
```

Comment:
I feel like here there should be a heading because there is a big jump. Maybe: 
```suggestion

## Introduction

`KeptnAppContext` is a custom resource in Keptn that allows you to add metadata
```